### PR TITLE
[13.0] Migrate product_packaging_type to 13.0

### PR DESCRIPTION
**Pull request by guewen**
Port of #525

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by emagdalenaC2i** [comment]
Is it a migration or a new module?
I did not find the previous product_packaging_type

File: product_packaging_type/models/product_packaging_type.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            msg = _('There must be one product packaging type set as "Is Default".')
+        elif default_count > 1:
+            msg = _('Only one product packaging type can be set as "Is Default".')
+        if msg:
+            raise ValidationError(msg)
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+    _order = "product_id, type_sequence"
```
Comment (jbaudoux) [inline product_packaging_type/models/product_packaging_type.py:35]: I would order by default based on the contained quantity
```suggestion
    _order = "product_id, qty, type_sequence"
```

Comment (guewen) [reply product_packaging_type/models/product_packaging_type.py:35]: It makes sense, but then I wonder if the `type_sequence` is really useful?

**Comment by guewen** [comment]
@emagdalenaC2i link for the previous version is in the PR's description (it's an unmerged PR).

File: product_packaging_type/models/product_packaging_type.py (hunk 1 above)
Comment (simahawk) [reply product_packaging_type/models/product_packaging_type.py:35]: @jbaudoux ping :)

**Comment by rousseldenis** [comment]
@guewen @simahawk Beta/Alpha module does not mean no tests, no convention...

Could you add tests and split models per file please ? Thanks

**Pull request merged**